Resources/Prototypes/Datasets/Names/fortunes.yml

```
@@ -1,82 +1,82 @@
-  - Take a quick look in the mirror!
-  - Don't look in the mirror today!
-  - Friendship will come to you soon.
-  - A grey man in a gas mask will come to you soon.
-  - Whoever gets this note will receive a chocolate bar.
-  - Milk is good for your bones.
-  - Shout "Hurray!" and get insulated gloves.
-  - Hug your captain and you will receive a gift.
-  - A dubious friend may be an enemy in red camouflage.
-  - A faithful colleague is a strong defense.
-  - A friend asks only for your time, not your crowbar.
-  - Your life doesn't get better by chance, it gets better by doing your job properly.
-  - Boo!
-  - The real treasure is the friends you made along the shift.
-  - Allan please add details
-  - You are the greatest person who ever lived!
-  - NanoTrasen is forcing me to make fortunes for these cookies, please help!
-  - Buy Mr. Chang's today!
-  - You love Mr. Chang!
-  - Don't buy Discount Dan. Buy Mr. Chang!
-  - Don't buy Getmore Chocolate. Buy Mr. Chang!
-  - What’s hidden in a maintenance locker?
-  - All your hard work will soon pay off.
-  - Don’t just think, act!
-  - An apple a day keeps the doctor away.
-  - Don't drink and set up singulo.
-  - Stare into the singulo, and the singulo stares back at you.
-  - Every good and perfect gift is from CentCom.
-  - There’s no such thing as Space Station 13.
-  - There is a time for caution, but not for fear.
-  - The smart thing to do is to begin trusting your intuitions.
-  - Time is precious, but insulated gloves are more precious than time.
-  - Fortune is a giver and a taker. Do it.
-  - Fortune and flowers do not last forever. Do it.
-  - Do it. Just do it.
-  - When life gives you lemons instead of oranges, make lemonade.
-  - By opening this cookie you’ve set a terrible chain of events in motion, good job.
-  - Get back to work!
-  - Cookie.print() not found
-  - THERE IS NO CLONER ON THIS STATION
-  - Putting suit sensors on max is always a good idea.
-  - Syndicate is hiring. Contact us.
-  - The person next to you is up to something. You know what you have to do.
-  - Did you know? Monkeys are not what they seem.
-  - Did you know that half of salvage specialists go missing on the job?
-  - Also try out sashimi with galaxythystle sauce!
-  - Did you know that 70% of Nanotransen applicants go missing within the first years of their contract? The more you know!!
-  - This shift you are going to do a very robust move in front of everyone.
-  - Robustness is not just about how good you are, but how good of a person you are.
+  - Конец близок... и это всё ТВОЯ вина.
+  - Мы знаем что ты сделал.
+  - Остерегайся.
+  - Не оборачивайся.
+  - Никогда не доверяй персоне в скрипучей обуви.
+  - Никогда не доверяй персоне в красной униформе.
+  - Никогда не доверяй персоне в зеленой униформе.
+  - Никогда не доверяй персоне в синей униформе.
+  - Никогда не доверяй персоне в желтой униформе.
+  - Никогда не доверяй персоне в черной униформе.
+  - Никогда не доверяй персоне без униформы
```

Просто униформы или одежды?
```suggestion
  - Никогда не доверяй персоне без униформы.
```

---

одежды fixed